enterprise/backend/src/metabase_enterprise/serialization/cmd.clj

```diff
@@ -47,6 +48,7 @@
                        context)]
     (try
       (do
+        (log/infof "BEGIN LOAD from %s with context %s" path context)
```

Comment:
You should probably i18n INFOs. For DEBUG and TRACE I think no i18n is fine since people don't really enable those unless someone (usually us) is debugging something

Reply:
bump

Reply:
Sorry, forgot about this one.  Should be fixed now.